### PR TITLE
frontend: leverage theme palette for input components

### PR DESCRIPTION
**Pull request by lucechal14**
Replace hardcoded colors with theme palette references for components in input

Testing Performed
manual, unit tests

### Before (left) vs After (right)

**Checkbox Panel**
Screenshot 2023-12-07 at 2 58 18 p m

**Checkbox**
Screenshot 2023-12-07 at 2 58 45 p m

**DateTimePicker**
Screenshot 2023-12-07 at 2 59 26 p m

**Form**
Screenshot 2023-12-07 at 3 00 29 p m

**MultiSelect**
Screenshot 2023-12-07 at 3 00 57 p m

**RadioGroup**
Screenshot 2023-12-07 at 3 01 29 p m

**Radio**
Screenshot 2023-12-07 at 3 01 57 p m

**Select**
Screenshot 2023-12-07 at 3 02 20 p m

**Switch**
Screenshot 2023-12-07 at 3 02 45 p m

**TextField**
Screenshot 2023-12-07 at 3 03 05 p m

**TimePicker**
Screenshot 2023-12-07 at 3 03 30 p m

**ToggleButtonGroup**
Screenshot 2023-12-07 at 3 03 55 p m

### CODE REVIEW

File: frontend/packages/core/src/Assets/emojis/happy.tsx [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
+        width="28"
+        height="14"
+      >
+        <path
+          d="M19.8261 35.4783C27.3179 35.4783 33.3913 29.8721 33.3913 22.9565H6.26086C6.26086 29.8721 12.3342 35.4783 19.8261 35.4783Z"
+          fill="#0D1030"
+        />
+      </mask>
+      <g mask="url(#mask0_76_9148)">
+        <circle cx="19.8261" cy="35.4782" r="11.3043" fill={theme.palette.secondary[600]} />
```
[the remark this answers is not in the document]
Comment (jdslaugh) [reply frontend/packages/core/src/Assets/emojis/happy.tsx:38]: Yeah I agree, all assets/emojis should be as is in their SVG, so no need to update them

**Review by jdslaugh — COMMENTED** [review]
So one thing we should start doing, especially for future PR's and probably this one, is to upload before/after screenshots of each component. This can be from storybook or from the app itself. This would allow us to see the changes a little clearer since reviewing it I can only go off of the syntax and making sure its correct, versus checking the styling.

File: frontend/packages/core/src/Assets/emojis/happy.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (lucechal14) [reply frontend/packages/core/src/Assets/emojis/happy.tsx]: Change reverted

**Review by jecr — APPROVED** [review]

**Review by jdslaugh — APPROVED** [review]
Thanks!

**Pull request merged**